deeppavlov/utils/server/server.py

```diff
@@ -31,7 +32,18 @@
 
 SERVER_CONFIG_FILENAME = 'server_config.json'
 
+
+class PollerFilter(Filter):
+    pat = re.compile(r'POST\s/\S*/poller\s')
+    def filter(self, record):
+        if PollerFilter.pat.search(record.getMessage()):
+            return False
+        return True
```

Comment:
`return not PollerFilter.pat.search(record.getMessage())`?

Reply:
done